CHANGES.md

```diff
@@ -15,6 +15,7 @@
 ### :bug: Bugs fixed
 
 - Fix pencil loader [Taiga #8348](https://tree.taiga.io/project/penpot/issue/8348)
+- Fix can't collapse colors and typographies groups when searching in the assets tab [Taiga #8125](https://tree.taiga.io/project/penpot/issue/8125)
```

Comment:
If this PR is against develop the changes should be in 2.2.0, if those changes are for 2.1.1 then it should be merged to staging

Reply:
Also, it's strange that a reference to Taiga #8125 appears twice in the CHANGES.md, or me it would make more sense pointing the PR to staging and not including more changes in the CHANGES.md file

Reply:
Fixed